src/components/UserList/index.tsx

```diff
@@ -66,10 +70,31 @@ const UserList: React.FC = () => {
         appearance: 'error',
       });
     } finally {
+      setDeleting(false);
       revalidate();
     }
   };
 
+  const importFromPlex = async () => {
+    setImporting(true);
+
+    try {
+      await axios.post('/api/v1/user/import-from-plex');
+      addToast(intl.formatMessage(messages.importedfromplex), {
+        autoDismiss: true,
+        appearance: 'success',
+      });
```

Comment:
What about having the toast say the # of new users that were added? `4 new users imported from Plex` for example.